dxp/demystifying_aui_input_tags.md

```diff
@@ -0,0 +1,727 @@
+# AUI Input
+
+## Table of Contents
+
+<div><a class="f5" href="#user-content-text">Text</a></div>
+<div><a class="f5" href="#user-content-textarea">Textarea</a></div>
+<div><a class="f5" href="#user-content-range">Range</a></div>
+
+The `<aui:input />` taglib is a custom tag used in `jsp` files to generate HTML form elements that conform to Liferay Portal's markup patterns, provide language translations, and other convenience methods. We should use this when adding form elements inside `jsp` files.
+
+The only required attribute is `name`, without it the `jsp` build will fail. The example below shows the minimum amount of code required for the taglib to output markup.
+
+**⚠️ Counterexample:**
```

Comment:
How is this one a counterexample? 😂

Reply:
Do you like this counter example?

![counter-example](https://user-images.githubusercontent.com/7074/95089140-135cd480-0724-11eb-9e28-febeae1ed60c.jpg)

Reply:
> How is this one a counterexample? 😂

I was thinking we should always supply a `type` since there is so much magic.